sdks/python/apache_beam/testing/util.py

```diff
@@ -105,3 +111,12 @@ def default_label(self):
       return label
 
   actual | AssertThat()  # pylint: disable=expression-not-assigned
+
+
+def open_shards(glob_pattern):
+  """Returns a composite file of all shards matching the given glob pattern."""
+  with tempfile.NamedTemporaryFile(delete=False) as f:
+    for shard in glob.glob(glob_pattern):
+      f.write(file(shard).read())
+    concatenated_file_name = f.name
+  return file(concatenated_file_name, 'rb')
```

Comment:
This is an issue for all tests at the moment.  I have filed https://issues.apache.org/jira/browse/BEAM-2413.